### ISSUE TITLE
Heading levels should only increase by one -  #solutions-deploy-app,solution-details-panel,calcite-panel  > .flex-direction-column.display-flex  > h3

### ISSUE THREAD

**Issue by MikeTschudi** [Esri/solutions-components#1631]
### Violation Details:

**Rule:** heading-order
**Rule Name:** Ensure the order of headings is semantically correct
**Description:**  Heading levels should only increase by one
**Standard:**  Deque Best Practices
**WCAG Criteria:**  Deque Best Practices
**Violations:**  1
**Impact:**  Moderate

 https://dequeuniversity.com/rules/axe/4.10/heading-order?application=axeAPI

- **Element Location:** #solutions-deploy-app,solution-details-panel,calcite-panel  > .flex-direction-column.display-flex  > h3
- **Element Source:** "< h3 class="headings-font" >< !--?lit$511213563$-- >Screenshots< /h3 >"
- **Suggested Fix:** "Fix any of the following:  Heading order invalid"
- **Screenshot Reference:** https://desktopqa/psregression/Accessibility/-855148073_heading-order.png
